framework/src/timeintegrators/ActuallyExplicitEuler.C

```diff
@@ -95,6 +95,8 @@ ActuallyExplicitEuler::solve()
   *_nonlinear_implicit_system->solution = _nl.solutionOld();
   *_nonlinear_implicit_system->solution += _solution_update;
 
+  _nl.overwriteContact(*_nonlinear_implicit_system->solution);
```

Comment:
Comment here would be appreciated about why this is needed. Also could we re-name this method? Maybe something like `overwriteNodeFace`? Of course that still begs the question of why node-face is special. Perhaps the added comment would explain why contact/node-face is special

Reply:
Added a comment and renamed it. The reason why we are doing something like that for contact with explicit dynamics is that we are solving a surrogate of the problem in an uncoupled way. Once that's done, we can 'preset' or 'overwrite' whatever values the displacement or velocity values had.